subjects/views.py

```diff
@@ -82,7 +82,7 @@ class SearchForm(forms.Form):
 # Args: [sub_id] - id of the given node to browse [sub_name] - name of the given
 # node to browse
 def search_subjects(request):
-    query_string = ''
+    # query_string = ''
```

Comment:
Rather than commenting this line out, let's just delete it. Same for next change.